### PR TITLE
[MFTF] Refactoring of AdminCreateOrderAddProductCheckboxTest

### LINKED ISSUE #31033: [Issue] [MFTF] Refactoring of AdminCreateOrderAddProductCheckboxTest
This issue is automatically created based on existing pull request: magento/magento2#31016: [MFTF] Refactoring of AdminCreateOrderAddProductCheckboxTest

---------
### Description (*)
The test is refactored according to the best practices followed by MFTF.

### Manual testing scenarios (*)
1. Check a product to be added to the Order 
2. Check if "Select" checkbox is checked in the Products Grid
3. Click the "Add selected product(s) to order" button

### PR DESCRIPTION
**Pull request by AnnaAPak**
### Description (*)
The test is refactored according to the best practices followed by MFTF.

### Manual testing scenarios (*)
1. Check a product to be added to the Order 
2. Check if "Select" checkbox is checked in the Products Grid
3. Click the "Add selected product(s) to order" button

### Resolved issues:
1. [x] resolves magento/magento2#31033: [MFTF] Refactoring of AdminCreateOrderAddProductCheckboxTest

### CODE REVIEW

**Comment by AnnaAPak** [comment]
@magento run Functional Tests CE, Functional Tests EE, Functional Tests B2B

File: app/code/Magento/Sales/Test/Mftf/ActionGroup/AddSimpleProductToOrderAndCheckCheckboxActionGroup.xml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AdminAddSimpleProductToOrderAndCheckCheckboxActionGroup" extends="AddSimpleProductToOrderActionGroup">
+        <annotations>
+            <description>Adds the provided Simple Product to an Order. Checks if checkbox is checked. Fills in the provided Product Qty. Clicks on 'Add Selected Product(s) to Order'.</description>
+        </annotations>
+
+        <seeCheckboxIsChecked selector="{{AdminOrderFormItemsSection.rowCheck('1')}}" stepKey="verifyProductChecked" after="selectProduct"/>
```
Comment (rogyar) [inline app/code/Magento/Sales/Test/Mftf/ActionGroup/AddSimpleProductToOrderAndCheckCheckboxActionGroup.xml:16]: Hi @AnnaAPak. Thank you for your work. Please, correct me if I'm wrong, but this action group is supposed to add a product to the order. But I see only the `seeCheckboxIsChecked` assertion in the action group.

**Review by rogyar — CHANGES_REQUESTED** [review]

File: app/code/Magento/Sales/Test/Mftf/ActionGroup/AddSimpleProductToOrderAndCheckCheckboxActionGroup.xml (hunk 1 above)
Comment (AnnaAPak) [reply app/code/Magento/Sales/Test/Mftf/ActionGroup/AddSimpleProductToOrderAndCheckCheckboxActionGroup.xml:16]: Hi @rogyar. Yes, it extends `AddSimpleProductToOrderActionGroup` - so both product adding and assertion are performed. Thanks.

**Comment by gabrieldagama** [comment]
@magento create issue

File: app/code/Magento/Sales/Test/Mftf/ActionGroup/AddSimpleProductToOrderAndCheckCheckboxActionGroup.xml (hunk 1 above)
Comment (rogyar) [reply app/code/Magento/Sales/Test/Mftf/ActionGroup/AddSimpleProductToOrderAndCheckCheckboxActionGroup.xml:16]: Oh, I see, missed the "extend" part. Thanks.

**Comment by rogyar** [comment]
@magento run Functional Tests CE, Functional Tests EE, Functional Tests B2B

**Review by rogyar — APPROVED** [review]
The failing tests are not related to the current changeset

**Comment by engcom-Alfa** [comment]
QA not applicable

**Comment by engcom-Foxtrot** [comment]
@magento run all tests

**Comment by engcom-Foxtrot** [comment]
@magento run all tests

**Comment by engcom-Foxtrot** [comment]
@magento run all tests

**Pull request merged**